packages/babel-plugin-transform-react-constant-elements/README.md

```diff
@@ -55,6 +66,22 @@ npm install --save-dev babel-plugin-transform-react-constant-elements
 }
 ```
 
+#### Options
```

Comment:
Can we change this to:

```
## Options

### `allowMutablePropsOnTags`

`Array<string>`, defaults to `[]`

If you are using a particular library (like `react-intl`) that uses object properties, and you are sure
that the element won't modify its own props, you can whitelist the element so that objects are allowed.

This will skip the `Mutable Properties` deopt.

<example>
```

(will improve website heading nav)